src/register.js

```diff
@@ -17,6 +17,7 @@ class DocgenPanel extends React.Component {
   }
 
   onAddDocs(docgenInfo) {
+    console.log(docgenInfo)
```

Comment:
Is this intentional?

Reply:
Nope, I should have taken that out. I need to make a point of looking at the diff before pushing..
Thanks!

Reply:
I'm not sure if you use `git add -p` or not but I would recommend trying it if you aren't because I find it helps to catch things like this.

Reply:
I will make a point of that, thank you.